runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/GradleDokkaSourceSetBuilderExtensions.kt

```diff
@@ -13,7 +12,7 @@ fun GradleDokkaSourceSetBuilder.dependsOn(sourceSet: KotlinSourceSet) {
 /**
  * Convenient override to **append** source sets to [GradleDokkaSourceSetBuilder.dependentSourceSets]
  */
-fun GradleDokkaSourceSetBuilder.dependsOn(sourceSet: AndroidSourceSet) {
+fun GradleDokkaSourceSetBuilder.dependsOn(@Suppress("DEPRECATION") sourceSet: com.android.build.gradle.api.AndroidSourceSet) {
```

Comment:
Should this function be deprecated as well, and a new function created for the replacement? Or what's the story behind the deprecation of `AndroidSourceSet`?

Reply:
`com.android.build.gradle.api.AndroidSourceSet ` is deprecated with replacement `com.android.build.api.dsl.AndroidSourceSet` (it is marked `@Incubating`).
But `com.android.build.gradle.api.AndroidSourceSet: com.android.build.api.dsl.AndroidSourceSet` 
 
 Anyway, I have created a new function for `com.android.build.api.dsl.AndroidSourceSet`. I am not sure that somebody does not use `@Incubating` API.